examples/README.md

```diff
@@ -45,10 +45,8 @@ Each example consists of two modules to define its behavior:
 ```js
 import * as pc from 'playcanvas';
 
-const canvas = document.getElementById('application-canvas');
-if (!(canvas instanceof HTMLCanvasElement)) {
-    throw new Error('No canvas found');
-}
+const canvas = /** @type {HTMLCanvasElement} */ (document.getElementById('application-canvas'));
```

Comment:
Why the outer brackets?

Reply:
Mantadory for jsdoc type casting

Reply:
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#casts

Reply:
Can you put the JSDoc on the line above?

Reply:
OK, I take it back. You can't. 👍

Reply:
No casting can only be inline since the jsdoc type is refering to the rhs of the expression not the left